mobile/mobile-core/devices-service.ts

```diff
@@ -672,6 +686,17 @@ export class DevicesService extends EventEmitter implements Mobile.IDevicesServi
 		shouldReturnImmediateResult = shouldReturnImmediateResult || false;
 		return { shouldReturnImmediateResult: shouldReturnImmediateResult, platform: platform };
 	}
+
+	private getEmulatorError(error: Error, platform: string): string {
+		let emulatorName = "Emulator";
+		if (this.$mobileHelper.isiOSPlatform(platform)) {
+			emulatorName = "Simulator";
+		}
+
+		return `Cannot find connected devices.${EOL}` +
+			`${emulatorName} start failed with: ${error.message}${EOL}` +
+			`To list currently connected devices and verify that the specified identifier exists, run '${this.$staticConfig.CLIENT_NAME.toLowerCase()} device'.`;
```

Comment:
I think it will be good to add t`ns devices --available-devices` as it is not visible inside specific platform. Moreover as `tns devices` lists active devices for both platform, and `tns devices ios ` for specific platform it seems consistent that both should accept `--available-devices`. @rosen-vladimirov what do you think?